clients/core-java/src/main/java/org/commonjava/indy/client/core/IndyClientHttp.java

```diff
@@ -317,10 +314,14 @@ public HttpResources getRaw( final String path, final Map<String, String> header
     {
         connect();
 
-        CloseableHttpResponse response = null;
+        CloseableHttpResponse response;
```

Comment:
See: https://stackoverflow.com/questions/18707676/how-to-set-the-default-header-for-all-requests-in-apache-http-client#21352161

Reply:
Ok

Reply:
merged.